docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "3.7"
+services:
+  # This runs the database that everything else connects to
+  postgres-db:
+    image: postgres:11.10
+    environment:
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD:-postgres}
+      - POSTGRES_USER=${POSTGRES_USER:-postgres}
+      - POSTGRES_DB=${POSTGRES_DB:-postgres}
+    volumes:
+      - db-volume:/var/lib/postgresql/data
+    ports:
+      - "5432:5432"
+    restart: unless-stopped
+    networks:
+      - default
+    command: >
+      postgres -c logging_collector=on -c log_destination=stderr -c
+      log_directory=/var/lib/postgresql/data/logs -c log_rotation_age=60 -c
+      log_truncate_on_rotation=on -c log_filename=server_log.hour.%H%M
```

Comment:
Also, why this command? :)

Reply:
this command I just copy and pasted from [another project (a nuxtjs variant of graphile/starter)](https://github.com/JoeSchr/graphile-starter) I'm using a postgres db docker-compose.yml as well. I figured it didn't hurt me, so why not use it. Honestly I think I just got this from upstream...

Reply:
Really cool to see someone in the crypto space who also works with graphile :)